README.md

```diff
@@ -1,24 +1,112 @@
+> - see the money spent on each category.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- Boostrap
+- Postgres SQL
+- Rspec
+- Capybara
 
-* Database creation
+## Live Demo (if available)
 
-* Database initialization
+[live demo](https://morris-coin-app.herokuapp.com/)
 
-* How to run the test suite
+## Getting Started
 
-* Services (job queues, cache servers, search engines, etc.)
+To get a local copy up and running follow these simple example steps.
 
-* Deployment instructions
+### Prerequisites
 
-* ...
+- You should have ruby installed in your machine, you can follow the steps given by the [official documentation](https://www.ruby-lang.org/en/documentation/installation/).
+
+- You should have a postgres user with superuser permissions. you can refer to the official [postgres documentation](https://www.postgresql.org/docs/current/role-attributes.html#:~:text=To%20create%20a%20new%20database,that%20is%20already%20a%20superuser.&text=A%20role%20must%20be%20explicitly,use%20CREATE%20ROLE%20name%20CREATEDB%20.) to create or update a role.
+
+### Setup
+
+First, you must clone this repository locally running this command:
+
+```
+git clone git@github.com:marurevi/Budget-app.git
+```
+
+and navigate to the cloned directory:
+
+```
+cd budget_app
+```
+
+### Install
+
+Once inside the project directory, you should install all project dependencies:
+
+- Install bundler:
+
+```
+gem install bundler
+```
+
+- Install project dependencies
+
+```
+bundle install
+```
+
+### Usage
+
+You can get the app running locally by opening a local server:
+
+```
+bin/rails server
+```
+
+### Run tests
+
+If you're on a linux-based system, you could run the executable file to run all specs:
+
+```
+bin/rspec
+```
+
+Windows users have to specify ruby in this case:
+
+```
+ruby bin/rspec
+```
+
+## Authors
+
+👤 **Menanya Morris**
+
+- GitHub: [@emmykage](https://github.com/Emmykage)
+- Twitter: [@mennydev](https://twitter.com/mennydev)
+- LinkedIn: [Menanya Morris](https://www.linkedin.com/in/morris-menanya)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). Published under the [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/).
+- Linters workflow created by [@anagudelogu](https://github.com/anagudelogu)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide instructions below it.
Implementing the above change(s) will make your ReadMe stand out even better 😉